lib/elliptic/utils.js

```diff
@@ -37,7 +38,6 @@ function toArray(msg, enc) {
   }
   return res;
 }
-utils.toArray = toArray;
```

Comment:
Oops, just noticed I exported on `exports` not `utils` as before. I pushed another commit.